sourcing/domain/service.go

```diff
@@ -115,6 +116,7 @@ func (d *DefaultSourcingService) GetAvailableSources(ctx context.Context, produc
 
 	sources, err := d.availableSourcesProvider.GetPossibleSources(ctx, product, deliveryInfo)
 	if err != nil {
+		d.logger.Warn("no possible sources available: " + err.Error())
```

Comment:
let's not log here, if the outer layer wants to log they can use the error that we bubble up to do so..